src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java

```diff
@@ -135,4 +135,20 @@ public void testConvertToStandardHtmlPart() throws Exception {
         
         //other cases are tested in testComparison
     }
+    
+    @Test
+    public void testReplacement() throws IOException {
```

Comment:
I'm asking to ensure 'minimal and sufficient documentation' for god mode. It doesn't have to be as detailed as the gist. Dev docs need to describe how to use God mode. Code comments should be enough for future maintainers of God mode code. I consider Gists as temporary notes we created to facilitate the discussion, and as such, not part of long-term documentation.